packages/types/src/amount.ts

```diff
@@ -1,15 +1,24 @@
 import { Amount } from '@buf/penumbra-zone_penumbra.bufbuild_es/penumbra/core/num/v1alpha1/num_pb';
 import { fromBaseUnit, joinLoHi, splitLoHi } from './lo-hi';
 import BigNumber from 'bignumber.js';
+import { DenomMetadata } from '@buf/penumbra-zone_penumbra.bufbuild_es/penumbra/core/asset/v1alpha1/asset_pb';
+import { getDisplayDenomExponent } from './denom-metadata';
 
 export const joinLoHiAmount = (amount: Amount): bigint => {
   return joinLoHi(amount.lo, amount.hi);
 };
 
-export const fromBaseUnitAmount = (amount: Amount, exponent: number): BigNumber => {
+export const fromBaseUnitAmount = (amount: Amount, exponent = 0): BigNumber => {
```

Comment:
Since we're occasionally passing in an empty `DenomMetadata` upstream of this call, I'll leave the default exponent as `0`, so that we're just multiplying by `1`.

Reply:
@grod220 hmm yeah, possibly. I think the case where we'd have missing denomMetadata is when we have a transaction but we don't have denom metadata for its asset type — which, if I understand correctly how Penumbra works, should never happen, right? In which case you're right that we should throw (somewhere in `fetchers/balances.ts`).